test/unit/adapters/sequelize.test.js

```diff
@@ -454,6 +454,45 @@ describe('=> Sequelize adapter', () => {
       { title: 'Tyrael', count: 1 },
       { title: 'Leah', count: 2 },
     ]);
+
+    result = await Post.findAll({
+      attributes: 'count(*) AS count',
+      group: [ 'title' ],
+      order: [[ 'title', 'desc' ]],
+    });
+    assert.deepEqual(result, [
+      { title: 'Tyrael', count: 1 },
+      { title: 'Leah', count: 2 },
+    ]);
+  });
+
+  it('Model.findAll({ having })', async () => {
+    await Promise.all([
+      { title: 'Leah' },
+      { title: 'Leah' },
+      { title: 'Tyrael' },
+    ].map(opts => Post.create(opts)));
+
+    let result = await Post.findAll({
+      attributes: 'count(*) AS count',
+      group: 'title',
+      order: [[ 'title', 'desc' ]],
+      having: 'count(*) = 2'
+    });
+    assert.deepEqual(result, [
+      { title: 'Leah', count: 2 },
+    ]);
+
+    result = await Post.findAll({
+      attributes: 'count(*) AS count',
+      group: [ 'title' ],
+      order: [[ 'title', 'desc' ]],
+      having: raw('count(*) = 2')
+    });
+
+    assert.deepEqual(result, [
+      { title: 'Leah', count: 2 },
+    ]);
```

Comment:
same as above, I think breaking into separated test cases helps regression tests and debugging

Reply:
`having([ 'count = ?', 2 ])` should be supported as well, which can be achieved with `spell.$having(...having)`